mySQL/posts.php

```diff
@@ -0,0 +1,12 @@
+<?php
+    include("config/init.php");
+
+    function getAllBlogPosts(){
```

Comment:
You have another function elsewhere with this same name. It looks like this one isn't used anywhere. No problem, but you can probably delete this file if you want just to keep your code base cleaner.